doc/source/faq.rst

```diff
@@ -82,15 +82,15 @@ because GCC's gcov tool does not make the necessary information available,
 and because different projects are interested in different kinds of branches.
 However, gcovr has the following options to *reduce* unwanted branches:
 
-With the :option:`gcovr --exclude-unreachable-branches` option,
-gcovr parses the *source code* to see whether that line even contains any code.
+With the :option:`--exclude-unreachable-branches <gcovr --exclude-unreachable-branches>`
```

Comment:
Is the ` <gcovr --exclude-unreachable-branches>` needed? I used it always without it and see no difference.

Reply:
`:option:` works like other sphinx roles for cross-referencing: the `<foo>` part is the target, it is necessary if the text you want to display is different from the target. You can omit it if you don't need to override the displayed text. In this case I wanted to omit the "gcovr" part from the displayed text because it seemed redundant, and also for consistency with the rest of the documentation where the vast majority of options are referenced without "gcovr".

Or are you saying that `:option:--exclude-unreachable-branches` should work by itself, without "gcovr" at all?

Reply:
Yes, this is how it's done in other parts of the documentation. But this needs an explicit `.. program:: gcovr` in the file.

Reply:
Ok I see, I can do it that way if you prefer.

Reply:
It's less to write and I think easier to read in the source. Would be good if it's aligned in the whole documentation.

Reply:
It's less to write and I think easier to read in the source. Would be good if it's aligned in the whole documentation.

Reply:
agreed.